### DISCUSSION TITLE
How to show larger images on fullscreen?

### DISCUSSION THREAD

**Discussion by theo-staizen** [igordanchenko/yet-another-react-lightbox#312] [General]
Hi all,

We are using the carousel as inline and we have the fullscreen plugin enabled.

Currently the images we are loading, are optimized for the inline carousel which is 600x600px, however when going fullscreen, we want to allow users to look at the full quality image and enable zooming in (we are using the zoom plugin).

Is it possible to pass different image urls to use for fullscreen?
Do we need to build a custom slide component (and will it work with zoom plugin?)

**Comment by igordanchenko** [comment]
Please see slide `srcSet` prop - https://yet-another-react-lightbox.com/documentation#Slide
